### ISSUE TITLE
include dataset Ohio State Acarology Laboratory (OSAL), Ohio State University  

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/globalbioticinteractions#445]
suggested by @neilcobb via https://github.com/globalbioticinteractions/globalbioticinteractions/issues/444#issuecomment-573828249 :

``` this dataset has lots of associations if you have not mined it yet https://www.gbif.org/dataset/96b54e8c-f762-11e1-a439-00145eb45e9a ```

Referred dataset points to IPT resource http://xbiod.osu.edu/ipt/archive.do?r=osal . And, that resource has been down for over a year (see https://hash-archive.org/history/http://xbiod.osu.edu/ipt/archive.do?r=osal ).

**Comment by jhpoelen** [comment]
@neilcobb - would you happen to know if Hans Klompen of OSU is aware of the IPT being down?

PS Note that I am not surprised by the broken link on GBIF - see figure 2. of preprint https://doi.org/10.32942/osf.io/mysfp .

**Comment by neilcobb** [comment]
No, and I assume that Norm Johnson did it for them and his IPT has been down for over a year, I stopped asking him when it would be functional.

**Comment by jhpoelen** [comment]
Good to know. Is there any tool on the GBIF side that help inform the curator of a dataset that their IPT is down? If so, can you point me to it?

**Comment by neilcobb** [comment]
I just write the helpdesk.  But my guess is that OSU knows it is down.  I am perplexed as to why they can solve the problem.
